zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1970,6 +1970,64 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /authorizations/search:
+    post:
+      tags:
+        - Authorization
+      summary: "Query authorizations (alpha)"
+      description: |
+        Search for authorizations based on given criteria.
+
+        :::note
+        This endpoint is an alpha feature and not enabled on Camunda clusters out of the box.
+        See the [Camunda 8 REST API overview](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md#query-api)
+        for further details.
+        :::
+      operationId: "findAllAuthorizations"
```

Comment:
🔧 For consistency with the other search endpoints, can we name this `findAuthorizations`? 🙂